argoverse/utils/sim2.py

```diff
@@ -35,6 +35,24 @@ def __init__(self, R: np.ndarray, t: np.ndarray, s: Union[int, float]) -> None:
         self.t_ = t.astype(np.float32)
         self.s_ = float(s)
 
+    @property
+    def theta_deg(self) -> float:
+        """Recover the rotation angle `theta` (in degrees) from the 2d rotation matrix.
+
+        Note: the first column of the rotation matrix R provides sine and cosine of theta,
+            since R is encoded as [c,-s]
+                                  [s, c]
+
+        We use the following identity: tan(theta) = s/c = (opp/hyp) / (adj/hyp) = opp/adj
```

Comment:
Nit: period and end of sentence for consistency.

Reply:
Missing return statement in the docstring.

Reply:
Regarding the return statement: I don't think it would add much, except bloat the function.

The Google python style guide says https://google.github.io/styleguide/pyguide.html#383-functions-and-methods:

```
A function must have a docstring, unless it meets all of the following criteria:

not externally visible
very short
obvious
```
I believe these 3 criteria are met here, so I believe it's unnecessary to have the "Returns" section

Reply:
OKay, good to know that we are following some set standards.